src/Interface/InterfaceView/InterfaceView.js

```diff
@@ -133,9 +155,14 @@ export const InterfaceView = ({ item = {}, isNarrow = false }) => {
   );
 };
 
+InterfaceView.manifest = {
+  interfaceCredentials: interfaceCredentialsResource,
+};
+
 InterfaceView.propTypes = {
   item: PropTypes.object.isRequired,
+  resources: PropTypes.object.isRequired,
   isNarrow: PropTypes.bool,
 };
 
-export default InterfaceView;
+export default withRouter(stripesConnect(InterfaceView));
```

Comment:
and view should be connected with anything because it's just view

Reply:
done